Snowflake.Data/Core/FileTransfer/EncryptionProvider.cs

```diff
@@ -25,6 +29,8 @@ class EncryptionProvider
         // The default block size for AES
         private const int AES_BLOCK_SIZE = 128;
         private const int blockSize = AES_BLOCK_SIZE / 8;  // in bytes
+        private const int OneMegabyteInBytes = 1024 * 1024;
```

Comment:
Can we add as a connection option? default can be 1 MB. It's trade of space v/s time.

Reply:
I added as a parameter to the connection string.